src/Resources/config/no_driver_admin.php

```diff
@@ -0,0 +1,18 @@
+<?php
+
+declare(strict_types=1);
+
+/*
+ * This file is part of the Sonata Project package.
+ *
+ * (c) Thomas Rabaix <thomas.rabaix@sonata-project.org>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+use Symfony\Component\DependencyInjection\Loader\Configurator\ContainerConfigurator;
+
+return static function (ContainerConfigurator $containerConfigurator): void {
+    $containerConfigurator->services();
```

Comment:
I dont think this line is needed

Reply:
In MediaBundle exists some block which require Admin services. 
IMO will be nice to allow register admin service with `manager_type: no_driver`.
https://docs.sonata-project.org/projects/SonataAdminBundle/en/4.x/getting_started/creating_an_admin/#step-2-register-the-admin-class 

Then in `no_driver_admin.php` we should add this admins. 

`no_driver` should display Admin panel with alert about missing driver.

Reply:
I dont understand what you mean. 

I think there is a bug about this file missing. But the line I mention it is not needed. 

The other thing you mention is maybe a feature for 3.x. Not for 4.x.

Reply:
I mean we should integrate `no_driver` in AdminBundle too. Then we should register admin services in `no_driver_admin.php`.

Agree. Services should be add in 3.x